css/style.css

```diff
@@ -1,19 +1,123 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
-body {
+/* Body */
+body h1{
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
```

Comment:
This tag [doesn't look to be a standard one](https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth) - any specific reason you used it here?